cnf-certification-test/identifiers/identifiers.go

```diff
@@ -908,6 +909,22 @@ tag. (2) It does not have any of the following prefixes: default, openshift-, is
 		},
 		TagCommon)
 
+	TestOperatorHasSemanticVersioningIdentifier = AddCatalogEntry(
+		"installed-operator-semantic-versioning",
+		common.OperatorTestKey,
+		`Tests whether an application Operator has semantic versioning.`,
```

Comment:
```suggestion
		`Tests whether an application Operator's semantic versioning is correct`,
```

Reply:
Following this, I propose something else : `Tests whether the operator's version used semantic versioning` https://sdk.operatorframework.io/docs/best-practices/best-practices/